app/src/main/java/org/sopt/sample/presentation/common/GlideModule.kt

```diff
@@ -0,0 +1,8 @@
+package org.sopt.sample.presentation.common
+
+import com.bumptech.glide.annotation.GlideModule
+import com.bumptech.glide.module.AppGlideModule
+
+@GlideModule
+class GlideModule : AppGlideModule() {
+}
```

Comment:
```suggestion
class GlideModule : AppGlideModule() 
```

Body가 필요없으면 굳이 안적어도 될 것 같아요